be/src/exec/pipeline/operator.h

```diff
@@ -118,12 +118,19 @@ class Operator {
     // equal to ExecNode::eval_join_runtime_filters, is used to apply bloom-filters to Operators.
     void eval_runtime_bloom_filters(vectorized::Chunk* chunk);
 
+    // 1. (-∞, s_pseudo_plan_node_id_upper_bound] is for operator which is not in the query's plan
+    // for example, LocalExchangeSinkOperator, LocalExchangeSourceOperator
+    // 2. (s_pseudo_plan_node_id_upper_bound, -1] is for operator which is in the query's plan
+    // for example, ResultSink
+    static const int32_t s_pseudo_plan_node_id_for_result_sink;
```

Comment:
Why not use the real plan node id?

Reply:
some operators like `LocalExchangeSourceOperator` which is not in the query plan do not have real plan node id, and these operators are added in addition to adjust the degree of parallelism